src/PowerShellEditorServices/Services/DebugAdapter/Handlers/ConfigurationDoneHandler.cs

```diff
@@ -156,23 +156,19 @@ private static PSCommand BuildPSCommandFromArguments(string command, IReadOnlyLi
                 return new PSCommand().AddCommand(command);
             }
 
-            // We are forced to use a hack here so that we can reuse PowerShell's parameter binding
+            // HACK: We use AddScript instead of AddArgument/AddParameter to reuse Powershell parameter binding logic.
+            // We quote the command parameter so that expressions can still be used in the arguments.
             var sb = new StringBuilder()
                 .Append("& ")
-                .Append(StringEscaping.SingleQuoteAndEscape(command));
+                .Append('"')
```

Comment:
Should these be <kbd>'</kbd> or are we purposefully trying to enable variable expansion?

Reply:
Purposefully allowing expressions to close another issue. Since arbitrary code can be specified anyways single quoting doesnt buy us anything.